src/rubymotion-apple.h

```diff
@@ -29,6 +29,7 @@ VALUE rb_uint2big(VALUE);
 long rb_num2int(VALUE);
 #define FIX2INT(x) ((int)FIX2LONG(x))
 #define NUM2INT(x) (FIXNUM_P(x) ? FIX2INT(x) : rb_num2int((VALUE)x))
+#define SSIZET2NUM(v) LONG2NUM(v)
```

Comment:
Oh very good point, I wasn't aware of that! Let's definitely use the first version then :)